ops/docs/update_docs.sh

```
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+if [[ -n "$(git status --porcelain -- . ':(exclude)ops/docs')" ]]; then
+  echo "Working tree has non-docs changes. Commit or stash them before running docs update."
+  exit 1
+fi
+
+LAST_COMMIT_HASH="$(git rev-parse --short HEAD)"
```

Shellcheck (0.11.0)

[warning] LAST_COMMIT_HASH appears unused. Verify use (or export if used externally).

(SC2034)